commit 6b8766080d2c331a05abbddbf3c7332dbb9df791
Author: Jeremy Reizenstein <reizenstein@meta.com>
Date:   Tue Dec 5 03:15:02 2023 -0800

    Use cuda's make_float3 in pulsar
    
    Summary: Fixes github.com/facebookresearch/pytorch3d/issues/1680
    
    Reviewed By: MichaelRamamonjisoa
    
    Differential Revision: D51587889
    
    fbshipit-source-id: e68ae32d7041fb9ea3e981cf2bde47f947a41ca2

diff --git a/pytorch3d/csrc/pulsar/global.h b/pytorch3d/csrc/pulsar/global.h
index a5b7fcb..b92afba 100644
--- a/pytorch3d/csrc/pulsar/global.h
+++ b/pytorch3d/csrc/pulsar/global.h
@@ -56,6 +56,7 @@
 #pragma clang diagnostic pop
 #ifdef WITH_CUDA
 #include <ATen/cuda/CUDAContext.h>
+#include <vector_functions.h>
 #else
 #ifndef cudaStream_t
 typedef void* cudaStream_t;
@@ -74,13 +75,6 @@ struct float3 {
 };
 #endif
 namespace py = pybind11;
-inline float3 make_float3(const float& x, const float& y, const float& z) {
-  float3 res;
-  res.x = x;
-  res.y = y;
-  res.z = z;
-  return res;
-}
 
 inline bool operator==(const float3& a, const float3& b) {
   return a.x == b.x && a.y == b.y && a.z == b.z;
